packages/flyout/src/Flyout.js

```diff
@@ -194,15 +194,21 @@ export default class Flyout extends Component {
   }
 
   handleChildMouseEnter = () => {
+    const { props } = this.props.children;
     if (this.props.openOnHover) {
       this.setOpen(true);
+      return props.onMouseEnter();
```

Comment:
i don't think this is the correct condition to put this in. if the child does not have an `onMouseEnter`, i think this will error out.

Reply:
What do you think 
``if(this.props.openOnhover && props.onMouseEnter())``
or maybe i need to relocate this return

Reply:
it should probably be independent of `openOnHover`. i don't know if the `return` out is necessary but usually the pattern the hig has followed is if the event handler exists then call it.

Reply:
@wmui51  changes applied